src/Acme/Bundle/EnrichBundle/Resources/config/processors.yml

```diff
@@ -0,0 +1,10 @@
+parameters:
+    pim_enrich.mass_edit.capitalize_values.processor.class: Acme\Bundle\EnrichBundle\Processor\MassEdit\CapitalizeValuesProcessor
```

Comment:
I think you can directly put the class name without using the parameter for integration purpose but as you want ;)